packages/source-fetcher/package.json

```diff
@@ -26,7 +26,6 @@
   },
   "types": "dist/index.d.ts",
   "dependencies": {
-    "@types/node": "^15.0.1",
```

Comment:
This package's tsconfig [references](https://github.com/trufflesuite/truffle/blob/develop/packages/source-fetcher/tsconfig.json#L21) this. Make sure you can remove it there before saying it's not a dependency.

Although it probably should be a devDep.

Reply:
I'm not sure what the intention was here. According to the [docs](https://www.typescriptlang.org/tsconfig#typeRoots)

> If typeRoots is specified, **_only_** packages under typeRoots will be included.  

[This config](https://github.com/trufflesuite/truffle/blob/develop/packages/source-fetcher/tsconfig.json#L17-L21) implies that `node` is excluded because it wouldn't appear in `typeRoots` list of paths

```
    "typeRoots": [
      "./typings",
      "../../**/node_modules/@types/debug"
    ],
    "types": ["node"]
```
      
Have you used any tools to verify this behavior? I'm gonna spend an hr with [tsd](https://github.com/SamVerschueren/tsd) to see if it can help.

Reply:
Just try taking it out and see if it breaks

Reply:
Yeah you should just see if this builds. This is a package that is not "required" and used really so you can't really tell if it is necessary until you clear out all `node_modules` and try to build. If TS doesn't complain afterwards, you are probably good.

Reply:
Will people be trying to install this package into their projects and building with TS? :)